task/ike-create/0.1/README.md

```diff
@@ -0,0 +1,39 @@
+# Istio Workspace Create Task
+
+`ike-create` creates a new `session` object for `istio-workspace` and will provide a private route in the cluster for the given target.
+
+## `ike-create`
+
+### Parameters
+
+-   `namespace`(string): Which namespace to target.
+
+    -   default value: $(context.taskRun.namespace)
+
+-   `target`(string): What deployment to target.
+
+-   `session`(string): What to call the session.
+
+-   `route`(string): How to route.
+
+    -   default value:
+
+-   `image`(string): The image to use for the route.
+
+### Results
```

Comment:
@vinamra28 did we decide on Results+workspace or either ?

Reply:
I guess we can continue with both `Results+workspace` for now